src/core/extension.ts

```diff
@@ -132,10 +134,14 @@ export class BallerinaExtension {
                 log(`Plugin version: ${pluginVersion}\nBallerina version: ${ballerinaVersion}`);
                 this.checkCompatibleVersion(pluginVersion, ballerinaVersion);
 
-                // versions less than 1.1.0 are incapable of handling cli commands for langserver and debug-adapter
-                this.isNewCLICmdSupported = this.compareVersions(ballerinaVersion, "1.1.0", true) >= 0;
-                // versions higher than 1.2.0 are not accepting cli commands parameters
-                this.isNewConfigChangeSupported = this.compareVersions(ballerinaVersion, "1.2.0", true) >= 0;
+                if (ballerinaVersion.match(SWAN_LAKE_REGEX)) {
+                    this.isNewConfigChangeSupported = true;
```

Comment:
Shouldn't we set `isNewCLICmdSupported` as well?

Reply:
Fixed with https://github.com/ballerina-platform/plugin-vscode/pull/169/commits/cd2112fbfdf3e6e14ad8ec7efae2bf3c6fd98cbe